_editor.cfg

```diff
@@ -49,11 +44,6 @@
         append=yes
         title= _ "FtF - No Way Out"
     [/music]
-    [music]
-        name=zhaytee-wesnoth_tragedy.ogg
-        append=yes
-        title= _ "FtF - Wesnoth Tragedy"
-    [/music]
```

Comment:
likewise, why remove this?

Reply:
as with the previous one, never mind, I see now that you changed it to pull from UMC Book 10 instead...